### PR TITLE
Added entitiesAreLoaded method to entitiesCollection

### PR DESCRIPTION
**Pull request by hAdamm**

### CODE REVIEW

File: src/Relationships/EntitiesCollection.php [hunk 1]
```diff
@@ -73,12 +73,19 @@ public function isDeletingInProgress() {
...
+    }
+
+    /**
      * @return string
      */
     public function getEntitiesClassName() {
         return $this->relationShip->entityClassNameToBeConnected;
     }
-
+
```
Comment (lordjim666) [inline src/Relationships/EntitiesCollection.php]: whitespace?

File: tests/TestCases/EntitiesCollection/EntitiesAreLoadedTest.phpt [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+    public function testFullAccessToCollection() {
+        $contact = $this->getContact()->save();
+        $contact->phones->asArray();
+        \Tester\Assert::true($contact->phones->getEntitiesAreLoaded());
+    }
+
+    public function createDefaultDataStorage(\obo\Relationships\EntitiesCollection $phonesCollection) {
+        return  \Mockery::mock(new \obo\Tests\Assets\DataStorage);
+
```
Comment (lordjim666) [inline tests/TestCases/EntitiesCollection/EntitiesAreLoadedTest.phpt]: blank line

File: src/Relationships/EntitiesCollection.php (hunk 1 above)
Comment (hAdamm) [reply src/Relationships/EntitiesCollection.php]: fixed

File: tests/TestCases/EntitiesCollection/EntitiesAreLoadedTest.phpt (hunk 2 above)
Comment (hAdamm) [reply tests/TestCases/EntitiesCollection/EntitiesAreLoadedTest.phpt]: fixed

**Pull request merged**